react-components/src/architecture/base/commands/BaseInputCommand.ts

```diff
@@ -5,42 +5,34 @@ import { type TranslationInput } from '../utilities/TranslateInput';
 import { RenderTargetCommand } from './RenderTargetCommand';
 
 export abstract class BaseInputCommand extends RenderTargetCommand {
-  protected _placeholder?: TranslationInput;
-  protected _content?: string;
-  protected _okButtonLabel?: TranslationInput;
-
-  protected _onFinish?: () => void;
```

Comment:
The fields `placeholder`, `okButtonLabel` and `onFinish` turned out to be unused/unneeded in this context, so they have been removed altogether